src/site/content/en/blog/performance_tool_score_diffs.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Performance Tooling Score Differences
+subhead: I get different scores in Test My Site, PageSpee Insights, and Lighthouse - why?
+authors:
+  - egsweeny
+date: 2019-07-05
+hero: hero.jpg
+# You can adjust the position of your hero image with this property.
+# Values: top | bottom | center (default)
+# hero_position: bottom
+alt: A description of the hero image for screen reader users.
+description: |
+  This post serves to explain the reasons that there are differences in performance scores between Test My Site (TMS), PageSpeed Insights (PSI), and Lighthouse. This document does not detail other differences in the tools, such as server run locations, network throttling presets, etc.
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - performance
+---
+
+# Why is my performance score different between different tools? What is my REAL score?
```

Comment:
I'd suggest removing this heading as it might look a little weird when the page first renders. Usually we rely on the title acting as the h1.